### ISSUE TITLE
Quest bug: [They Grow Up So Fast] (Alliance)

### ISSUE THREAD

**Issue by Haltus-Kain** [gamefreedomgit/Maelstrom#5525]
Description:
Kalimdor | Winterspring | Quest | [[They Grow Up So Fast] (Alliance)](https://warcraft.wiki.gg/wiki/They_Grow_Up_So_Fast_(Alliance))

Quest cannot be turned in despite having the required items in inventory.

How to reproduce:
Acquire:
[[Winterspring Cub Whisker]](https://warcraft.wiki.gg/wiki/Winterspring_Cub_Whisker) x 20
[[Runecloth]](https://warcraft.wiki.gg/wiki/Runecloth) x 20
[[Rugged Leather]](https://warcraft.wiki.gg/wiki/Rugged_Leather) x 20
[[Winterspring Cub]](https://warcraft.wiki.gg/wiki/Winterspring_Cub_(quest_item)) (provided)
80 gold

Attempt to turn in quest.

Playing on a Human Death Knight named "Haltus".

How it should work:
With the above items in inventory, quest should be able to be completed.

![image](https://github.com/gamefreedomgit/Maelstrom/assets/87287845/7d48ccde-f9e2-4473-a39b-a77f05d66072)

**Comment by Haltus-Kain** [comment]
> cannot reproduce, i have created the same class,race,nickname and tried to complete:

I'll revisit next time I'm on and check back - there's been a couple updates since starting this one, maybe it fixed itself.

I have a hypothesis though: this and many of the other bugged quests I'm finding might have something to do with the DK starting zone skip. I've seen other players working on quests that I can't access like the arena in Twilight Highlands: I've done all the quests there except the arena, but it still doesn't show. (don't think I've reported that one yet - on the list!!)

I used the little ghost girl to skip the DK zone; could that have flagged something as incomplete that's acting as a prerequisite for some other random quests in the open world?

Happy to help science it if you want to pull me back into the DK starting phase, and I can run through those normally and see if anything changes after. If yes and if you're able to move/phase me while I'm offline, go ahead whenever it's convenient and I'll check back with the results.
